shelter/remoteattestation/ra.h

```diff
@@ -18,34 +18,7 @@ static const int rsa_pub_3072_pcks_header_len = 24;
 static const int rsa_pub_3072_raw_der_len = 398; /* rsa_pub_3072_pcks_der_len - pcks_nr_1_header_len */
 
 typedef struct {
-    uint8_t quote[2048];
+    uint8_t quote[8192];
```

Comment:
Please define 8192 as a contsant rather than hard coding.

Reply:
You can see the code style complying with the below definition in the same file in ra.h. So i suggest to keep this one quote[8192] without change.
typedef struct {
    uint8_t ias_report[2*1024];
    uint32_t ias_report_len;
    uint8_t ias_sign_ca_cert[2*1024];
    uint32_t ias_sign_ca_cert_len;
    uint8_t ias_sign_cert[2*1024];
    uint32_t ias_sign_cert_len;
    uint8_t ias_report_signature[2*1024];
    uint32_t ias_report_signature_len;
} attestation_verification_report_t;

Reply:
> You can see the code style complying with the below definition in the same file in ra.h. So i suggest to keep this one quote[8192] without change.
> typedef struct {
> uint8_t ias_report[2_1024]; uint32_t ias_report_len; uint8_t ias_sign_ca_cert[2_1024];
> uint32_t ias_sign_ca_cert_len;
> uint8_t ias_sign_cert[2_1024]; uint32_t ias_sign_cert_len; uint8_t ias_report_signature[2_1024];
> uint32_t ias_report_signature_len;
> } attestation_verification_report_t;

the 8192 is a not constant length but a max value for quote. When it change to a value more than 8192, a lot of hard coding must be revised. So it is better to define a Quote_Max_Length 8192 in your codes.

Reply:
> Quote_Max_Length

Besides, define a Quote_Max_Length can tell developers the quote length is not a constant but has a max value 8192. I think it is import.

Reply:
Actually, quote max length is not 8192. In dcap mode , it depends on its signature algorithm.  The Max will not be more than 6000. So keep it as a buffer length may be more reasonable instead of max length.

Reply:
> Actually, quote max length is not 8192. In dcap mode , it depends on its signature algorithm. The Max will not be more than 6000. So keep it as a buffer length may be more reasonable instead of max length.

Ok, what's your suggestions? @jiazhang0